src/OpenLoco/src/Localisation/StringIds.h

```diff
@@ -904,7 +904,7 @@ namespace OpenLoco::StringIds
     constexpr string_id station_cargo_en_route_start = 1212;
     constexpr string_id station_cargo_en_route_end = 1213;
     constexpr string_id no_space_for_more_vehicle_orders = 1214;
-
+    constexpr string_id tooManyOrdersForThisVehicle = 1215;
```

Comment:
was this a conscious decision to not snake_case this? I know this doesn't follow the convention but i've so far always done snake case.

Reply:
Yeah, few newer string ids I've been using lowerTitleCase instead. Not too fussed either way, to be honest. I think snake case is at least as legible for string id constants.